apps/HelloPyTorch/Makefile

```diff
@@ -0,0 +1,91 @@
+include ../support/Makefile.inc
+
+BUILD_DIR = build
+
+OPS = $(BIN)/$(HL_TARGET)/add_float32.a \
+	  $(BIN)/$(HL_TARGET)/add_float64.a \
+	  $(BIN)/$(HL_TARGET)/add_grad_float32.a \
+	  $(BIN)/$(HL_TARGET)/add_grad_float64.a
+
+# Check whether we have cuda installed, if add the CUDA ops as dependencies
+ifeq ($(shell which nvcc),)
+HAS_CUDA = 0
+CUDA_OPS =
+else
+HAS_CUDA = 1
+CUDA_OPS = $(BIN)/$(HL_TARGET)/add_cuda_float32.a \
+		   $(BIN)/$(HL_TARGET)/add_cuda_float64.a 
+endif
+
+all: test
+
+# Run the PyTorch tests to verify the module is compiled correctly
+test: .wrapper
```

Comment:
Is `.wrapper` a file that we are generating, or one coming from some external ecosystem? (If it's one of ours, that seems like an extremely generic extension and one we should improve. If it's external, nevermind, of course.)

Reply:
.wrapper is just a dummy file that is "touched" every time the `python setup.py` command is run, to avoid re-running it when nothing has changed (see https://github.com/halide/Halide/blob/f1aeaf8b1d9a51855d347a19c02d0c0db6ed298e/apps/HelloPyTorch/Makefile#L33). `setup.py` does not play well with Makefile dependencies, and the setup.py script generates several files whose name can be platform dependent, so I did not want to list their names here and in instead of the `.wrapper` target

Do you know of a better solution for this?